examples/offline_inference/disaggregated_prefill.py

```diff
@@ -0,0 +1,80 @@
+import os
+import time
+from multiprocessing import Event, Process
+
+from vllm import LLM, SamplingParams
+from vllm.config import KVTransferConfig
+
+
+def run_prefill(prefill_done):
+    os.environ["CUDA_VISIBLE_DEVICES"] = "0"
+
+    prompts = [
+        "Hello, my name is",
+        # "Hi, your name is", # To simulate transmission failure
+        "Tell me a very long story",
+    ]
+    sampling_params = SamplingParams(temperature=0, top_p=0.95, max_tokens=1)
+
+    ktc = KVTransferConfig.from_cli(
+        '{"kv_connector":"PyNcclConnector","kv_role":"kv_producer","kv_rank":0,"kv_parallel_size":2}'
+    )
+    llm = LLM(model="meta-llama/Meta-Llama-3.1-8B-Instruct",
+              kv_transfer_config=ktc,
+              max_model_len=2000,
+              gpu_memory_utilization=0.8)
```

Comment:
Some GPUs may get OOM with this ratio. Please document the GPU you used and advise the ratio for other GPUs.